src/terminal/ParseConfigFile.hpp

```diff
@@ -34,6 +34,8 @@
 #include <sys/time.h>
 #endif /* HAVE_SYS_TIME_H */
 
+#include "Headers.hpp"
```

Comment:
This was missing when cross-compiling to Android, but is reproducible on linux too if cotire is turned off. I guess precompiling the headers pulls in other headers that weren't included originally.